docs/adr/0009_controller_v2_lib_migration.md

```diff
@@ -0,0 +1,406 @@
+# OCM Kubernetes Controller Toolkit Migration Plan to new OCM Reference Library
+
+* **Status**: proposed
+* **Deciders**: OCM Technical Steering Committee
+* **Date**: 2025.09.23
+
+**Technical Story**:  
+Enable a clear and efficient migration path for engineers moving the OCM controllers (formerly ocm-k8s-toolkit) to the new reference library set maintained in the Open Component Model (OCM) monorepo.
+
+---
+
+## Context and Problem Statement
+
+The *ocm-k8s-toolkit* (now renamed to `open-component-model/kubernetes/controller`, referred to as the OCM Kubernetes Controller Toolkit) provides a set of controllers for managing OCM objects within Kubernetes clusters. Its primary responsibilities include:
+
+1. Synchronizing and verifying OCM Component Versions from OCM repositories.
+2. Aggregating and interpreting `.ocmconfig` configurations to enable repository access and configuration.
+3. Synchronizing and verifying OCM resources contained within OCM Component Versions.
```

Comment:
Feel like number 1/3 could be one sentence. Something like

1. Synchronizing and verifying OCM Component Versions and Resources from OCM repositories.